File: example/cs/helloworld/HelloWorld.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using BotPlatformSDKLib;

namespace botplatform.helloworld
{
    class HelloWorld
    {
        static void Main(string[] args)
        {
            RobotServerFactory serverFactory = new RobotServerFactory();
            serverFactory.Init(2);
            RobotServer server = serverFactory.CreateRobotServer("server.botplatform.com", 6602);
            server.MessageReceived += new _IRobotServerEvents_MessageReceivedEventHandler(server_MessageReceived);
            server.Login("your spid", "your sppwd", 60000);
            string cmd = null;
            while ((cmd = Console.ReadLine()) != null)
                if (cmd.Equals("exit")) break;
            server.Logout();
            serverFactory.Destroy();
        }
        static void server_MessageReceived(IRobotSession session, IRobotMessage message)
        {
            session.SendText("Hello World!");
            Console.WriteLine(message.Text);
        }
    }
}

File: example/vb/helloworld/helloworld.vb
Imports System
Imports System.Runtime.InteropServices
Imports BotPlatformSDKLib

Module helloworld

    Sub MessageReceived(ByVal session As IRobotSession, ByVal message As IRobotMessage)
        session.SendText("Hello World!")
        Console.WriteLine(message.Text)
    End Sub

    <MTAThread()> Sub Main()
        Dim robotServerFactory As RobotServerFactory = New RobotServerFactory
        robotServerFactory.Init(2)

        Dim robotServer As RobotServer = robotServerFactory.CreateRobotServer("bottest.com", 6602)
        AddHandler robotServer.MessageReceived, AddressOf MessageReceived
        robotServer.Login("SP000125", "test111", 60000)

        Dim cmd As String = ""
        Do While cmd <> "exit"
            cmd = Console.ReadLine()
        Loop

        robotServer.Logout()
        robotServerFactory.Destroy()
    End Sub

End Module
